sdks/python/apache_beam/testing/benchmarks/cloudml/cloudml_benchmark_test.py

```diff
@@ -55,7 +55,7 @@ def _publish_metrics(pipeline, metric_value, metrics_table, metric_name):
   )])
 
 
-@pytest.mark.uses_tft
+# @pytest.mark.uses_tft
```

Comment:
Can we just get rid of this comment (and the pytest import comment)

Reply:
Wait, don't we actually need this for https://github.com/apache/beam/blob/6cae195d4ab62a5b433f8781328ddc3f57b9c74f/sdks/python/test-suites/dataflow/common.gradle#L454

Reply:
Yes that is needed for testing, it won't be picked up in that suite if absent or commented out

Reply:
ah no. I will revert it once the expected test runs on Jenkins and is green

Reply:
Reverted the change.